### PR TITLE
3.7.5 release notes and knowledge center updates

### LINKED ISSUE #3316: v 3.7.5 Release Notes and Knowledge Center Updates
**Description:**
This ticket is to track release notes to STTs on the TDP portal, relative Knowledge Center Updates and the release notes Email sent to STTs.

**AC:**
- [x] Content drafted in Gitbook
- [x] Changes staged in Knowledge Center Test environment
- [x] A11y tested

**Tasks:**
- [x] UX Create a Ticket for Release Notes + Email Template 
- [x] UX/OFA Generate Draft in GitBook 
- [x] Receive Content Feedback from PM/Dev/UX/OFA, schedule sync to finalize 
- [x] UX Update release notes section and/or knowledge center for test environment
- [x] UX Update release number on release note provided by dev
- [x] Attach proof of a11y pass. 
- [x] 12/17: Update for additional section
- [x] UX Create a project repo PR and request final-mile approval from OFA. 
- [x] UX Makes final-mile edits to the PR/Test environment as needed until approved.
- [x] UX Create Email and send draft to Alex/Thomas for approval
- [x] UX makes edits to email as applicable, sends final draft
- [x] Dev cuts release
- [x] OFA approves PR
- [x] UX merges PR to Knowledge Center after release is pushed to production.
- [x] OFA sends email release notes to STTs

**Supporting Documentation and Links:**

- [Link to the Draft Content for Release Notes](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/release-notes-workflow/dec-4-ish-release-notes)
- [Test Environment with Release Notes](https://reitermb.github.io/Test/knowledge-center/index.html)
- [Test Environment KC: Viewing Error Reports](https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html)
- [Test Environment KC:Submission History](https://reitermb.github.io/Test/knowledge-center/view-submission-history.html)

### PR DESCRIPTION
**Pull request by victoriaatraft**
## Summary of Changes
Release notes and knowledge center updates
Pull request closes #3316 

## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.
> *Demo GIF(s) and screenshots for testing procedure*

### CODE REVIEW

File: product-updates/knowledge-center/viewing-error-reports.html [hunk 1]
```diff
@@ -693,6 +715,182 @@ <h4>File Pre-Check: Errors related to record length:</h4>
 
     <p>Please refer to the <a href="https://www.acf.hhs.gov/ofa/policy-guidance/final-tanf-ssp-moe-data-reporting-system-transmission-files-layouts-and-edits">Transmission File Layout documents</a> to compare your records against their expected layouts.</p>
 
+
+    <table class="usa-table">
```
Comment (ADPennington) [inline product-updates/knowledge-center/viewing-error-reports.html:719]: just noticed this addition. very nice 🚀

**Review by ADPennington — APPROVED** [review]
🥇

File: product-updates/knowledge-center/viewing-error-reports.html (hunk 1 above)
Comment (reitermb) [reply product-updates/knowledge-center/viewing-error-reports.html:719]: @ADPennington Noting a couple small tweaks I just committed, we were generating some a11y errors off of the table for empty primary columns (The original program type column). Changes:
- Eliminated that column by splitting the table up into three by program
- Replaced >= type notations with ≥ for better screenreader behavior

Resulting in: 
image


cc @victoriaatraft

**Comment by reitermb** [comment]
A11y scans: 
image
image
image

**Review by reitermb — APPROVED** [review]

**Review by klinkoberstar — APPROVED** [review]
lgtm!

**Comment by ADPennington** [comment]
@reitermb @victoriaatraft  will this merge soon?

**Comment by victoriaatraft** [comment]
Hi, sorry I thought this has been merged. I noticed on the [Gitbok process page](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/) that this step has not been assigned to anyone. 

Preference on who takes ownership of this task? It comes after Dev cuts the release, so I would think it would be dev? Happy to be wrong on that. I'm assuming it's selecting the "merge pull request" button? @ADPennington 
cc @reitermb @klinkoberstar @lhuxraft 

edit: i noticed on the list of tasks in the design ticket I had OFA merges PR to Knowledge Center after release is pushed to production, so i can definitely make that correction moving forward once we decide who takes ownership of that task....

next update: Andrew said I can press it and it's not scary anymore so I can assign it to UX once pr's get approved lol.

**Pull request merged**